x-pack/plugins/transform/public/app/sections/transform_management/components/search_selection/search_selection.tsx

```diff
@@ -64,6 +67,9 @@ export const SearchSelection: FC<SearchSelectionProps> = ({
                   defaultMessage: 'Saved search',
                 }
               ),
+              showSavedObject: (savedObject: SavedObject) =>
```

Comment:
Interestingly the transform wizard used to partially work when selecting an ES|QL based saved search, successfully pulling out the index pattern from the search to use in the transform. However the other commands in the search (WHERE, RENAME, DROP etc) would be ignored, so I think this change is correct, filtering out ES|QL based saved searches until if / when the transform wizard supports the whole of the search.